mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+go = "1.23"
```

Comment:
have we started using mise elsewhere? i'd rather drop(or ignore) if not the case yet.